credsweeper/app.py

```diff
@@ -104,25 +108,44 @@ def run(self, content_provider: FilesProvider) -> None:
             content_provider: path objects to scan
 
         """
-        file_extractors = []
-        if content_provider:
-            file_extractors = content_provider.get_scannable_files(self.config)
+        _empty_list_mypy_fix: List[TextContentProvider] = []
+        file_extractors: Union[List[DiffContentProvider], List[TextContentProvider]] = \
+            content_provider.get_scannable_files(self.config) if content_provider else _empty_list_mypy_fix
```

Comment:
Thank you for explanation. I understood what you mean.

But could you please change the name of `_empty_list_mypy_fix`? I think it would be good to change with `_empty_list`. How do you think?